src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
```

Comment:
Do you really need a flex container for the card ?

Reply:
> Do you really need a flex container for the card ?

no, but already used flex before and didn't want to change, because it works fine
now changed without flex

Reply:
done